cmd/metrics.go

```diff
@@ -517,7 +588,7 @@ func storageMetricsPrometheus(ch chan<- prometheus.Metric) {
 	// MinIO Total Disks per node
 	ch <- prometheus.MustNewConstMetric(
 		prometheus.NewDesc(
-			prometheus.BuildFQName("minio", "disks", "total"),
+			prometheus.BuildFQName(minioNamespace, "disks", "total"),
```

Comment:
On a 4 node (1 drive/node) cluster, `minio_disks_total` shows 1, but it should be 4. As we discussed, each node should show cluster level metrics.

![image](https://user-images.githubusercontent.com/5156139/103201875-06acde80-4917-11eb-9be7-bb614bfad65d.png)